diffkemp/simpll/Utils.cpp

```diff
@@ -86,7 +86,10 @@ std::string dropSuffix(std::string Name) {
 std::string getFileForFun(Function *Fun) {
     if (auto *SubProgram = Fun->getSubprogram()) {
         if (auto *File = SubProgram->getFile()) {
-            return File->getDirectory().str() + "/" + File->getFilename().str();
+            auto Directory = File->getDirectory();
```

Comment:
A similar change should be done to the `getSourceFilePath` function (it does essentially the same thing as `getFileForFun`, only for a scope instead of a function).